lib/matplotlib/axes/_base.py

```diff
@@ -1988,7 +1988,15 @@ def add_collection(self, collection, autolim=True):
             # Make sure viewLim is not stale (mostly to match
             # pre-lazy-autoscale behavior, which is not really better).
             self._unstale_viewLim()
-            self.update_datalim(collection.get_datalim(self.transData))
+            datalim = collection.get_datalim(self.transData)
+            points = datalim.get_points()
+            if not np.isinf(datalim.minpos).all():
+                # By definition, p0 <= minpos <= p1, so minpos would be
```

Comment:
what are p0 and p1 here?

Reply:
```suggestion
                # By definition, p0 <= minpos <= p1, so minpos (minimum positive value) would be
```

Reply:
Ah, oops, that made sense before the last commit which switched to `points`.